ext/reflection/php_reflection.c

```diff
@@ -2320,7 +2320,7 @@ ZEND_METHOD(ReflectionParameter, __construct)
 			break;
 
 		default:
-			zend_argument_error(reflection_exception_ptr, 1, "must be either a string, an array(class, method) or a callable object, %s given", zend_zval_type_name(reference));
+			zend_argument_error(reflection_exception_ptr, 1, "must be one of a string, an array(class, method), or a callable object, %s given", zend_zval_type_name(reference));
```

Comment:
```suggestion
			zend_argument_error(reflection_exception_ptr, 1, "must be a string, an array(class, method), or a callable object, %s given", zend_zval_type_name(reference));
```
Don't think "one of" really make sense in this context.

Reply:
Thanks for the feedback! I was hesitant with this one indeed. Will change it. :)